### PR TITLE
[KEYCLOAK-12439] Update to Angular 1.7.9

### PR DESCRIPTION
**Pull request by abstractj**
## [KEYCLOAK-12439](https://issues.redhat.com/browse/KEYCLOAK-12439)

Update to Angular 1.7.9

## Verification Steps

1. Remove `themes/src/main/node_modules/` and `themes/src/main/resources/theme/keycloak/common/resources/node_modules/` only to make sure that there's no cache
2. At the root folder of the Keycloak server repository run: 
```
mvn install -Pnpm-update -DskipTests=true && mvn clean install -Pdistribution -DskipTests=true
```
As alternative you can also run:
```
mvn -f testsuite/utils/pom.xml exec:java -Pkeycloak-server -Dkeycloak.profile.feature.account_api=enabled -Dkeycloak.profile.feature.account2=enabled -Dresources
```
4. Check if nothing was broken into the server
5. Run the integration tests as described in https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/HOW-TO-RUN.md#base-ui-tests.

## Additional Notes
* This PR introduces a ridiculous amount of changes, to be more precise **5,521** changed files. For this reason I made sure to split the commits. So people can focus on [what really matters](https://github.com/keycloak/keycloak/commit/2be4701d51f5ab0cce7a8470e20db44ba43dac9b).

### CODE REVIEW

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@abstractj I've very quickly checked your changes (I wanted to run the Admin Console tests) and the deps weren't really updated. You need to update [common/resources](https://github.com/keycloak/keycloak/tree/master/themes/src/main/resources/theme/keycloak/common/resources) dir too.

**Comment by edewit** [comment]
@abstractj I was expecting to have 1.7.9 here as well
image

**Comment by abstractj** [comment]
@edewit @vmuzikar thank you both for reviewing it, it should be ready to go now.

**Comment by edewit** [comment]
@abstractj I might do something wrong but I still get angular version 1.6.10 in the console

**Comment by vmuzikar** [comment]
@edewit @abstractj I've just briefly checked the latest changes and I can confirm Angular was updated in the Admin Console. Running Admin Console tests now.

**Comment by vmuzikar** [comment]
Btw, I think this PR conflicts with https://github.com/keycloak/keycloak/pull/6432.

**Comment by abstractj** [comment]
@vmuzikar I believe once that's merged I can do the updates to remove the conflicts. The only commit that truly matters for us is https://github.com/keycloak/keycloak/pull/6654/commits/46f4ffd7b56309eeef4b602290e65bb01fa74728.

If we get the approval tomorrow on Alex's PR, I can do the updates.

**Review by vmuzikar — DISMISSED** [review]
I've done a very brief review (basically just verified Angular.js was updated and Admin Console test passed). LGTM but we need to be careful about those conflicts. Thanks @abstractj!

**Comment by abstractj** [comment]
@vmuzikar @ssilvert @hmlnarik rebased after the merge of Alex's PR.

File: package-lock.json
Comment (vmuzikar) [inline package-lock.json]: @abstractj I'm not sure we should have a package-lock directly in the root directory (where's no node_modules). How did this even get here? :) However, it's a non-blocking issue.

File: themes/src/main/package-lock.json
Comment (vmuzikar) [inline themes/src/main/package-lock.json]: Again, non-blocking but I believe this package-lock shouldn't be here. If anywhere, should be in `themes/src/main/resources/theme/keycloak/common/resources` where the `package.json` lives.

**Review by vmuzikar — COMMENTED** [review]
Found two non-blocking issue (any of them doesn't affect functionality). Other than that LGTM, thanks @abstractj!

**Review by stianst — APPROVED** [review]

**Pull request merged**
